data/quarg/quarg missions.txt

```diff
@@ -274,3 +274,51 @@ mission "Quarg Ships Atrocity"
 	on offer
 		conversation "quarg imprisonment"
 
+
+
+mission "Quarg Pug Arfecta Warning"
+	minor
+	landing
+	source
+		attributes "quarg" "ringworld"
```

Comment:
Multiple attributes on the same line means "A or B". In this case, we want this to offer on a planet that is both Quarg AND a ringworld, in which case we use separate attribute lines.
```suggestion
		attributes "quarg"
		attributes "ringworld"
```